docs/coalton-lisp-interop.md

```diff
@@ -196,28 +196,28 @@ There are two ways to call Coalton from Lisp.
 The safe way to call Coalton is to use the `coalton` operator. This will type check, compile, and evaluate a Coalton expression and return its value to Lisp. Note that `coalton` does not accept definitions or top-level forms. For example:
 
 ```lisp
-CL-USER> (format t "~R" (coalton:coalton coalton-library::(length (cons 1 (cons 2 nil)))))
```

Comment:
This is not allowed behavior because of a package-lock on `coalton-prelude` (and `coalton`)